File: c/PROBLEMS/1182.c
#include <stdio.h>
#include <string.h>
int main(){
    double m[12][12], soma=0;
    int i, j, l;
    char s;

    scanf("%d %c", &l, &s);
    for (i=0;i<12;i++){
        for (j=0;j<12;j++){
            scanf("%lf", &m[i][j]);
        }
    }
    if (s == 'S') {
        i=l;
        while (i==l){
            for (j=0;j<12;j++){
                soma+=m[j][i];
            }
            i++;
        }
        printf("%.1lf\n", soma);
    } else if (s == 'M') {
        i=l;
        while (i==l){
            for (j=0;j<12;j++){
                soma+=m[j][i];
            }
            i++;
        }
        printf("%.1lf\n", soma/12);
    }
}

File: js/PROBLEMS/1182.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')
l = parseInt(lines.shift())
s = lines.shift()
m = []
for (i=0;i<12;i++){
    linha = []
    for (j=0;j<12;j++){
        linha.push(parseFloat(lines.shift()))
    }
    m.push(linha)
}

let soma = 0
for (let j=0;j<12;j++){
    soma += m[j][l]
}

if (s=='S'){
    console.log(soma.toFixed(1))
}else if (s=='M'){
    console.log((soma/12).toFixed(1))
}